docs/playbook.rst

```diff
@@ -77,3 +77,24 @@ When we will be done with the workshop, our final playbook will look something l
        - name: restart flask app
          service: name=flask-demo state=restarted
 
+We can also skip a particular task or make a task execute only if a condition is met using the When statement.
+
+.. code-block:: guess
```

Comment:
@containscafeine Can you check if you can put `yaml` as code-block so that syntax will be also highlight?